src/cmd/src/lib.rs

```diff
@@ -31,37 +31,39 @@ macro_rules! main {
         #[cfg_attr(coverage, no_coverage)]
         fn main() {
             let opts = clap::Parser::parse();
-            $crate::$component(opts);
+            let registry = prometheus::Registry::new();
```

Comment:
It seems the this registry is supposed to be a process-level singleton. Should we use a lazy init singleton to store it to avoid passing it in multiple places.

Reply:
Long long ago, we use lazy_static! for registry, but finally we remove it.
Let me investigate whether to add it back.